src/edu/stanford/nlp/sempre/Value.java

```diff
@@ -21,6 +21,9 @@
   @JsonValue
   public String toString() { return toLispTree().toString(); }
 
+  // (optional) Quick string used for sorting Values
+  public String quickString() { return toString(); }
```

Comment:
Why is this this needed if it just calls toString()?

Also, if its intended use is sorting values, then we should name it `sortString` or something more informative.